src/server/debugging.ts

```diff
@@ -210,6 +215,7 @@ function readProcessLine(line: string): RawProcessInfo {
         containerPID: Number(containerPID),
         containerPPID: Number(containerPPID),
         containerPGID: Number(containerPGID),
+        runState,
```

Comment:
As mentioned, I think it would be better to parse and validate the runState here